src/components/map/MapService.js

```diff
@@ -1503,12 +1508,15 @@ goog.require('ga_urlutils_service');
             }
           }).then(function(response) {
             var height = carto.height - response.data.height;
-            var offset = Math.tan(Cesium.Math.toRadians(50)) * (height + 50);
-            destination.x += offset * 2;
+            var offsetView = Math.tan(
+                Cesium.Math.toRadians(pitch + cameraFieldOfView / 2)) * height;
+            // Approx. direction on x and y (only valid for Swiss extent)
+            destination.x += offsetView;
+            destination.y += offsetView / 8;
```

Comment:
This is an approximation (like before), you need to move to the cartographic south, but here we are in Cartesian3 earth centered coordinate. In fact the offset view only gives you a magnitude.
Ideally we should have a unit vector in the direction we want and multiply it with our `offsetView` which more of a magnitude to get the translation vector of the positioned at the Nadir (no pitch).